api/controllers/CampaignBotController.js

```diff
@@ -36,7 +31,7 @@ class CampaignBotController {
   collectReportbackProperty(req, property, ask) {
     this.debug(req, `collectReportbackProperty:${property}`);
 
-    if (req.query.campaign || ask) {
+    if (req.query.campaign || ask || this.isKeywordCampaignBot(req)) {
```

Comment:
I think this was forward thinking to an issue about to create about making use of the `body.keyword` property we get from a Mobile Commons request, where we'll have different keyword types. Signup, Continue. CampaignBot keyword here should really be continue.

I'm thinking this new function could work like the `isCommand(req, type)` does -- `isKeyword(req, type)` where type can be `'signup'` or `'continue'`.